app/src/main/java/com/example/polina/socialnetwork/API.java

```diff
@@ -15,5 +15,7 @@
     public  JSONObject getProfile(Context context);
     public JSONObject newPost (Context context, String massage, JSONObject location,  String image, String account);
     public JSONObject getPosts(Context context, String id);
+    public JSONObject getLoadPosts(Context context, String idUser, String size, String idPost);
```

Comment:
please use getPosts and getLoadPosts as one method, rename paramenters to userId, postId, and size should be integer.